### PR TITLE
Added new networks

### PR DESCRIPTION
**Pull request by BellringerQuinn**

### CODE REVIEW

File: Packages/Sequence-Unity/Sequence/SequenceSDK/Ethereum/ChainDictionaries.cs [hunk 1]
```diff
@@ -48,6 +52,7 @@ public static class ChainDictionaries
             { Chain.OasysHomeverse, "OAS"},
             { Chain.AstarZKEvm, "ETH" },
             { Chain.Xai, "XAI" },
+            { Chain.Blast, "BLAST" },
```
Comment (taylanpince) [inline Packages/Sequence-Unity/Sequence/SequenceSDK/Ethereum/ChainDictionaries.cs:55]: Should be ETH

File: Packages/Sequence-Unity/Sequence/SequenceSDK/Ethereum/ChainDictionaries.cs [hunk 2]
```diff
@@ -59,7 +64,10 @@ public static class ChainDictionaries
             { Chain.TestnetPolygonAmoy, "MATIC" },
             { Chain.TestnetAstarZKyoto, "ETH"},
             { Chain.TestnetXrSepolia, "tXR" },
-            { Chain.TestnetXaiSepolia, "sXAI" }
+            { Chain.TestnetXaiSepolia, "sXAI" },
+            { Chain.TestnetB3Sepolia, "ETH" },
+            { Chain.TestnetAPEChain, "ETH" },
```
Comment (taylanpince) [inline Packages/Sequence-Unity/Sequence/SequenceSDK/Ethereum/ChainDictionaries.cs:69]: Should be APE

File: Packages/Sequence-Unity/Sequence/SequenceSDK/Ethereum/ChainDictionaries.cs [hunk 3]
```diff
@@ -59,7 +64,10 @@ public static class ChainDictionaries
...
+            { Chain.TestnetBlastSepolia, "BLAST" }
```
Comment (taylanpince) [inline Packages/Sequence-Unity/Sequence/SequenceSDK/Ethereum/ChainDictionaries.cs:70]: Should be ETH

File: Packages/Sequence-Unity/Sequence/SequenceSDK/Ethereum/ChainDictionaries.cs [hunk 4]
```diff
@@ -88,7 +97,10 @@ public static class ChainDictionaries
             { Chain.TestnetPolygonAmoy, "https://amoy.polygonscan.com/" },
             { Chain.TestnetAstarZKyoto, "https://astar-zkyoto.blockscout.com/" }, 
             { Chain.TestnetXrSepolia, "https://xr-sepolia-testnet.explorer.caldera.xyz/" },
-            { Chain.TestnetXaiSepolia, "https://testnet-explorer-v2.xai-chain.net/" } 
+            { Chain.TestnetXaiSepolia, "https://testnet-explorer-v2.xai-chain.net/" },
+            { Chain.TestnetB3Sepolia, "https://sepolia.explorer.b3.fun/" },
+            { Chain.TestnetAPEChain, "" }, // Not found
```
Comment (taylanpince) [inline Packages/Sequence-Unity/Sequence/SequenceSDK/Ethereum/ChainDictionaries.cs:102]: You can use https://curtis.explorer.caldera.xyz/

**Review by taylanpince — CHANGES_REQUESTED** [review]
Minor fixes

**Comment by BellringerQuinn** [comment]
Thanks for these @taylanpince!

**Review by taylanpince — APPROVED** [review]

**Pull request merged**